src/openklant/components/klantinteracties/api/tests/test_filters.py

```diff
@@ -92,6 +107,48 @@ def setUp(self):
             onderwerpobjectidentificator_code_register="5",
         )
 
+    def test_filter_partij_url(self):
+        partij_url = (
```

Comment:
I think using `reverse` here in combination with `http://testserver` would be nice.